api.md

```diff
@@ -469,7 +528,81 @@ CREATE INDEX ON conditions(time, location) USING brin
 ```
 
 ---
+## detach_data_node() [](detach_data_node)
+
+Detach a data node from one hypertable or from all hypertables.
+
+Reasons for detaching a data node:
 
+- A data node should no longer be used by a hypertable and needs to be
+removed from all hypertables that use it
+- You want to have fewer data nodes for a distributed hypertable to
+partition across to other distributed hypertables in the database
+
+>:WARNING: This function *MUST* be run on an access node.
```

Comment:
See my comment about the WARNING above.

Reply:
Removed for now, in lieu of a better solution